Servers/services/markdowns/complianceMarkdown.ts

```
@@ -31,10 +31,35 @@ type AllCompliances = ControlCategoryStructEUModel & {
 export async function getComplianceMarkdown (
     frameworkId: number,
     data: ReportBodyData
-  ) : Promise<any> {
-    let rows: string;
-    const reportData = await getComplianceReportQuery(frameworkId) as AllCompliances[];
-    
+  ) : Promise<String> {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)